workspaces/ai-integrations/plugins/ai-experience/src/components/TemplateSection/TemplateSection.tsx

```diff
@@ -39,15 +41,18 @@ export const TemplateSection = () => {
               link={`/create/templates/default/${item.metadata.name}`}
               title={item.metadata.title}
               description={item.metadata.description}
-              kind="Template"
+              kind={t('common.template')}
             />
           </Grid>
         ))}
       </Grid>
       <Box sx={{ pt: 2 }}>
         <ViewMoreLink to={catalogTemplatesLink} underline="always">
-          View all {templates?.totalItems ? templates?.totalItems : ''}{' '}
-          templates
+          {t('sections.viewAllTemplates' as any, {
+            count: templates?.totalItems
+              ? templates?.totalItems.toString()
+              : '',
+          })}
```

Comment:
Couldn't count handle the length / number?

Reply:
Yes, throws a type error for number.